base_tier_validation/__manifest__.py

```diff
@@ -1,14 +1,16 @@
 # Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
 {
     "name": "Base Tier Validation",
     "summary": "Implement a validation process based on tiers.",
-    "version": "12.0.3.1.0",
+    "version": "12.0.4.1.0",
     "development_status": "Mature",
     "maintainers": ['lreficent'],
     "category": "Tools",
     "website": "https://github.com/OCA/server-ux",
-    "author": "Eficent, Odoo Community Association (OCA)",
+    "author": "Eficent, Ecosoft, Odoo Community Association (OCA)",
```

Comment:
Don't need to add here. We are just doing enhancement .
But you can still add yourself as contributor in CONTRIBUTOR.rst